Resources/views/backend/config/view/form/document_paypal_unified.js

```diff
@@ -17,9 +17,21 @@ Ext.define('Shopware.apps.Config.view.form.DocumentPaypalUnified', {
      */
     getFormItems: function() {
         var me = this,
-            data = me.callParent(arguments);
+            formItems = me.callParent(arguments);
```

Comment:
Even though this plugin does not use our style guide, we should still apply our code style as much as possible. So please don't combine `var` declarations.